common-custom-user-data-maven-extension/CHANGELOG.md

```diff
@@ -0,0 +1,37 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+## [Unreleased]
```

Comment:
I fixed a bug that would not create a proper custom link.

Reply:
Added.